module/scripts/pych

```diff
@@ -60,9 +60,19 @@ def check(arg):
 
     # Check that Chapel libraries are there
     info(" * Libraries")
+    dir = os.sep.join([
+        CONFIG["compilers"]["chapel"]["root_path"],
+        CONFIG["compilers"]["chapel"]["lib_path"],
+    ])
+    if not os.path.exists(dir):
+        error( "Missing lib dir at (%s)" % (dir) )
+    else:
+        for file in ["libchpl.a", "main.o",]:   # FIXME: hard-coded in config/pych.json
+            if not os.path.exists(os.sep.join([dir, file,])):
+                error( "Missing Chapel lib file(%s) in dir(%s)" % (file, dir))
 
     # Check that c-headers are there
-    info(" * Headers")
+    # info(" * Headers")    # FIXME: should check for header files here
```

Comment:
Why did this get commented out?

Reply:
Because it makes you think that it has checked "Headers" when it actually hasn't done any such thing. Rather than stop and figure out what (if anything) it should try to check, and how to check it, I decided to just save that one for later. 

This script did not actually check "Chapel libraries" either, but I needed to implement that to make the installation instructions better.

Reply:
Oh, duh.  That's just an informational print, not a function call.  Cool, that's fine then